### PR TITLE
Course screen body layout

### PR DESCRIPTION
**Pull request by nunibaranes**
- Fixing `CourseScreen` (V2) max-height, added temporary calculation until the new header will be ready.
- Updating `wm-ui` version with fixes for `WMDrawer`.

### CODE REVIEW

File: src/components/Screen-v2/CourseScreen/index.tsx [hunk 1]
```diff
@@ -63,20 +63,24 @@ export default function CourseScreen(): ReactElement {
...
-              overview={overview as CourseOverviewData}
-              isLoading={isUpdating || isFetchingCourseData}
-            />
-          ),
-          destroyDrawerContentOnClose: true,
-        }}
-      >
-        <CourseTabs />
-      </WMDrawerLayout>
+      <div className={styles.screenBody}>
```
Comment (nunibaranes) [inline src/components/Screen-v2/CourseScreen/index.tsx:66]: Added Wrap for calculating the`screenBodyHeight`.

File: src/components/Screen-v2/CourseScreen/index.tsx [hunk 2]
```diff
@@ -63,20 +63,24 @@ export default function CourseScreen(): ReactElement {
...
+        <WMDrawerLayout
+          drawerProps={{
+            classes: { paper: styles.analyticsPaper },
```
Comment (nunibaranes) [inline src/components/Screen-v2/CourseScreen/index.tsx:69]: Added styles to drawer's paper.

File: src/components/Screen-v2/CoursesScreen/style.module.scss [hunk 3]
```diff
@@ -84,15 +82,15 @@
...
-
     --screenHeaderHeight: 102px;
     --screenBottomPadding: 5px;
     --screenBodyHeight: calc(
       100vh - var(--header-toolbar-height) - var(--screenHeaderHeight) - var(--screenBottomPadding)
     );
 
+    display: flex;
     height: var(--screenBodyHeight);
+    overflow: hidden;
```
Comment (nunibaranes) [inline src/components/Screen-v2/CoursesScreen/style.module.scss:93]: Preventing the appearance of the scroll bar

**Review by dr15 — APPROVED** [review]
Approved

**Pull request merged**